modelskill/plotting/_scatter.py

```diff
@@ -106,17 +107,21 @@ def scatter(
         x-label text on plot, by default None
     ylabel : str, optional
         y-label text on plot, by default None
-    skill_table: str, List[str], bool, optional
+    skill_table: str, List[str], dict[str,str], bool, optional
         calculate skill scores and show in box next to the plot,
         True will show default metrics, list of metrics will show
         these skill scores, by default False,
+        mapping can be used to rename the metrics in the table.
         Note: cannot be used together with skill_scores argument
     skill_scores : dict[str, float], optional
         dictionary with skill scores to be shown in box next to
         the plot, by default None
         Note: cannot be used together with skill_table argument
     skill_score_unit : str, optional
         unit for skill_scores, by default None
+    skill_score_names : dict[str, str], optional
```

Comment:
I don't think we should keep adding more arguments to `scatter`.
I think we should try to see if it is possible to attach this information to the metric object using the new @metric decorator introduced in #505